doc/testing-vendor-badges.md

```diff
@@ -0,0 +1 @@
+../vendor/README.md
```

Comment:
Ugh, I did not know that! 😕 Unfortunate, though good to know. Indeed, doesn't seem that important.